src/dpos/components/RegisterCandidateForm.vue

```diff
@@ -136,7 +137,11 @@ export default class RegisterCandidateForm extends Vue {
     }
     dposModule.registerCandidate(candidate)
   }
+
+  get validFee() {
+    return this.form.fee >= this.state.dpos.minCandidateFee
```

Comment:
`this.state.dpos.minCandidateFee` is in basis points, 100 == 1%, 1000 == 10% etc.
`this.form.fee` is not in basis points, it's in percent, so need to make an adjustment here.